apps/condo/domains/ticket/gql.js

```diff
@@ -7,19 +7,47 @@
 const { generateGqlQueries } = require('@condo/domains/common/utils/codegeneration/generate.gql')
 
 const COMMON_FIELDS = 'id dv sender v deletedAt newId createdBy { id name } updatedBy { id name } createdAt updatedAt'
+const COMMON_FILE_FIELDS = 'id file { id originalFilename publicUrl mimetype } organization { id } '
+const TICKET_FIELDS = `{ 
+    organization { id name } 
+    property { id name address } 
+    unitName 
+    entranceName 
+    floorName 
+    status { id name type organization { id } } 
+    statusReopenedCounter 
+    statusUpdatedAt 
+    statusReason 
+    number 
+    client { id name } 
+    clientName 
+    clientEmail 
+    clientPhone 
+    operator { id name } 
+    assignee { id name } 
+    executor { id name } 
+    watchers { id name } 
+    classifier { id name organization { id } parent { id name } } 
+    details 
+    related { id details } 
+    isEmergency 
+    isPaid 
+    meta 
+    source 
+    { id name type } 
+    sourceMeta 
+    ${COMMON_FIELDS} 
+}`
+const TICKET_STATUS_FIELDS = `{ organization { id } type name ${COMMON_FIELDS} }`
+const TICKET_FILE_FIELDS = `{  ticket { id } ${COMMON_FIELDS} ${COMMON_FILE_FIELDS} }`
+const TICKET_SOURCE_FIELDS = `{ organization { id } type name ${COMMON_FIELDS} }`
+const TICKET_CLASSIFIER_FIELDS = `{ organization { id } parent { id } fullName name ${COMMON_FIELDS} }`
 
-const TICKET_FIELDS = `{ organization { id name } property { id name address } unitName entranceName floorName status { id name type organization { id } } statusReopenedCounter statusUpdatedAt statusReason number client { id name } clientName clientEmail clientPhone operator { id name } assignee { id name } executor { id name } watchers { id name } classifier { id name organization { id } parent { id name } } details related { id details } isEmergency isPaid meta source { id name type } sourceMeta ${COMMON_FIELDS} }`
 const Ticket = generateGqlQueries('Ticket', TICKET_FIELDS)
-
-const TICKET_STATUS_FIELDS = `{ organization { id } type name ${COMMON_FIELDS} }`
```

Comment:
Please move the consts back. We needed to change the code generation template or don’t change the consts definition order

Reply:
Done